packages/astro/src/integration/create-integration.ts

```diff
@@ -36,6 +38,12 @@ function createIntegration<Params extends HotloadAstroClerkIntegrationParams>()
             logger.error('Invalid value for clerkJSVariant. Acceptable values are `"headless"`, `""`, and `undefined`');
           }
 
+          if (enableEnvSchema) {
+            logger.info(
+              'Env schema enabled. To ensure that secret keys are validated, please set `experimental.env.validateSecrets` to true.',
```

Comment:
By default, public variables are only validated on dev/build time. Private are validated at runtime only. Enabling this will also validate private vars on dev/build time